### ISSUE TITLE
Commenter failing on PNG images

### ISSUE THREAD

**Issue by romankaminski** [aquasecurity/tfsec-pr-commenter-action#42]
This is strange, as it was working just fine last week. No commiting PNG file to the repo (for documentation purposes) results in a following build failing:

```
+ echo 'tfsec violations were identified, running commenter...'
tfsec violations were identified, running commenter...
+ commenter
Starting the github commenter...
The commenter failed with the following error:
failed to create a new commenter. there were errors processing the PR files.
the patch details could not be resolved
```

Is there anything we can enable to debug this more?
We are using v0.1.10

**Comment by owenrumney** [comment]
I think this is when the patch details are calculated which wouldn't work with a png or other binary format. I will fix and update in the morning

**Comment by owenrumney** [comment]
@romankaminski  - I've fixed this in v1.0.1 of the action. it will gracefully step over binaries

**Comment by romankaminski** [comment]
Checked and it works. Thank you for quick reaction. Appreciated.
